.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,251 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
```

Comment:
you should normally not submit .idea files... but i guess that's because it's code styles?

Reply:
Sorry for this. All .idea folder has been added in .gitignore, but it didn't work for some reason.